x/programs/examples/pokemon_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package examples
+
+import (
+	"context"
+	_ "embed"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	//go:embed testdata/target/wasm32-wasi/release/pokemon.wasm
```

Comment:
The script should copy the wasm file into testdata/
```suggestion
	//go:embed testdata/pokemon.wasm
```